.travis.yml

```diff
@@ -1,24 +1,34 @@
-dist: trusty
+dist: xenial
 sudo: required
-cache: cargo
 language: rust
 addons:
   apt:
     packages:
+      - kcov
       - libcurl4-openssl-dev
       - libelf-dev
       - libdw-dev
       - binutils-dev
       - cmake
     sources:
       - kalakris-cmake
+      - sourceline: ppa:sivakov512/kcov
```

Comment:
I guess this is needed only for the new `kcov` package reference above to work, despite being added in a separate commit? In case the PPA gets broken, we can just remove these two lines, right?

I'm curious how you found the PPA, there's not a lot of Google hits for its name.

Reply:
> I guess this is needed only for the new kcov package reference above to work, despite being added in a separate commit? In case the PPA gets broken, we can just remove these two lines, right?

Yep, that's right. I screwed up a rebase, which is why this ended up in the wrong commit—but I've fixed that now—thanks! I've added some details (like "feel free to remove this PPA line if you can get a good version of kcov some other way") in a comment in the file.

> I'm curious how you found the PPA, there's not a lot of Google hits for its name.

Heh, I googled "kcov ppa" in the hopes that someone had already done the hard work of creating a PPA.